app/models.py

```diff
@@ -1145,8 +1127,6 @@ def _as_utils_template(self):
             return PlainTextEmailTemplate(self.__dict__)
         if self.template_type == SMS_TYPE:
             return SMSMessageTemplate(self.__dict__)
-        if self.template_type == BROADCAST_TYPE:
-            return SMSMessageTemplate(self.__dict__ | {"template_type": "sms"})
```

Comment:
Definitely no templates with this `template_type` in preview/staging/prod while not having a broadcasting service id? Didn't create any and then remove a service's broadcast permissions?